prow/tide/status.go

```diff
@@ -148,6 +148,15 @@ func requirementDiff(pr *PullRequest, q *config.TideQuery, cc contextChecker) (s
 		}
 	}
 
+	// Weight incorrect author with very high diff so that we select the query
+	// for the correct author.
+	if q.Author != "" && string(pr.Author.Login) != q.Author {
```

Comment:
Use `github.NormLogin` to normalize these before comparing (just strings.ToLower under the hood).